README.md

```diff
@@ -57,84 +50,37 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Personal portfolio] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+> Implementing HTML, CSS, GIT skills for making personal portfolio for mobile version
 
-**[your_project__name]** is a...
+**[Personal portfolio]** is file with personal record of your success or improvement in a range of areas.
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
+> Understand how to parse a Figma design to create a UI
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- **[figma template 1]**
+- **[HTML]**
+- **[CSS]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
+> Simply by cloning the repository folder in your local machine you can do any modification or explore thry this project. Good luck!
 
 To get a local copy up and running, follow these steps.
 
-### Prerequisites
-
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
 ### Setup
 
-Clone this repository to your desired folder:
+Clone this repository to your desired folder: https://github.com/TamaraDimov/Hello-world.git
 
```

Comment:
Great work @TamaraDimov for personalizing your README👍. Kindly correct the GitHub link to be cloned, as this is. portfolio project and not the 'hello world' that we currently have here.